build/templates/olm-artifacts-template.yaml.tmpl

```diff
@@ -185,6 +185,7 @@ objects:
         - routes
         verbs:
         - list
+        - get
```

Comment:
Why not watch as well? So it's consistent with the get, list, watch trio

Reply:
At this time the additions to dedicated-admins are based on specific needs and there's no specific request/need for watch at cluster scope.  If you look at the bits above this line you'll see a mix of list and get, the bare minimum RBAC to allow customer to see core projects in the console without a bunch of errors.